ui/app/AppLayouts/Chat/controls/community/CommunityCategoryListItem.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick 2.14
+
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
```

Comment:
does it make sense that this derives from `CommunityListItem`?

Reply:
Reusing `CommunityListItem` as a base was the easiest/fastest option. Creating new component instead of customization of that one would cause some code duplication presumably. Do you have other idea for that?

Reply:
Sorry, reading once again I see that my answer makes no sense, I read "... derives from  `StatusListItem`" :facepalm:. Indeed it makes sense, fixed!